### PR TITLE
Hide 'commit-next' from help messages

### LINKED ISSUE #1970: Make manual `confirma-activity` available only for debug
Or remove support for `nucypher ursula confirm-activity`

### PR DESCRIPTION
**Pull request by vzotova**
Fixes #1970

### CODE REVIEW

**Comment by KPrasch** [comment]
This raises a larger question: What role does the Ursula REPL play in nucypher?  

It was *once* an integral part of my workflow, but now I dont think we want to encourage running a worker as a non-service.  

I can imagine keeping this feature around for debugging situations (think get, IPFS console, etc.) however presenting it preeminently has been a source of UX confusion.

File: tests/acceptance/cli/ursula/test_ursula_command.py [hunk 1]
```diff
@@ -74,10 +74,28 @@ def write(data: bytes):
     with capture_output() as (out, err):
         protocol.lineReceived(line=b'bananas')
 
+    commands = protocol.commands
+    hidden_commands = ['?', 'commit_next']
```
Comment (fjarri) [inline tests/acceptance/cli/ursula/test_ursula_command.py:78]: Shouldn't we rather check for `commit-next` (with a hyphen)?

Comment (vzotova) [reply tests/acceptance/cli/ursula/test_ursula_command.py:78]: there were underscores in interactive console for commands, so I just did the same (in CLI still hyphen)

Comment (fjarri) [reply tests/acceptance/cli/ursula/test_ursula_command.py:78]: Ah, I see, I thought it was CLI.

**Pull request merged**